cmd/scollector/collectors/processes_linux.go

```diff
@@ -104,6 +110,8 @@ func linuxProcMonitor(w *WatchedProc, md *opentsdb.MultiDataPoint) error {
 		Add(md, "linux.proc.io_bytes", io[4], opentsdb.TagSet{"type": "read"}.Merge(tags), metadata.Counter, metadata.Bytes, descLinuxProcIoBytesRead)
 		Add(md, "linux.proc.io_bytes", io[5], opentsdb.TagSet{"type": "write"}.Merge(tags), metadata.Counter, metadata.Bytes, descLinuxProcIoBytesWrite)
 		Add(md, "linux.proc.num_fds", len(fds), tags, metadata.Gauge, metadata.Files, descLinuxProcFd)
+		Add(md, "linux.proc.start_time", start_ts, tags, metadata.Gauge, metadata.Timestamp, descLinuxProcStartTS)
+		Add(md, "linux.proc.uptime", timestamp-start_ts, tags, metadata.Gauge, metadata.Second, descLinuxProcUptime)
```

Comment:
This is not the uptime of the program. It is the uptime of the program since scollector has been running. I don't see the value of this metric, since one can always just subtract the start time and a timestamp to figure out uptime.